{
	"id": 119606768,
	"number": 13434,
	"state": "open",
	"title": "x/net/http2: TestServer_RejectsLargeFrames fails with \"An existing connection was forcibly closed by the remote host\" on windows",
	"body": "From windows-amd64-gce builder http://build.golang.org/log/a0906e4fa3713e6512cdff9af6662d1db0faad0d\r\n```\r\n--- FAIL: TestServer_RejectsLargeFrames (0.31s)\r\n\thttp2_test.go:64: 2015/11/30 20:57:53 127.0.0.1:49238: connection error: PROTOCOL_ERROR\r\n\thttp2_test.go:64: 2015/11/30 20:57:53 127.0.0.1:49238: connection error: PROTOCOL_ERROR\r\n\thttp2_test.go:64: 2015/11/30 20:57:53 127.0.0.1:49238: connection error: PROTOCOL_ERROR\r\n...\r\n\thttp2_test.go:64: 2015/11/30 20:57:53 127.0.0.1:49238: connection error: PROTOCOL_ERROR\r\n\thttp2_test.go:64: 2015/11/30 20:57:53 127.0.0.1:49238: connection error: PROTOCOL_ERROR\r\n\thttp2_test.go:64: 2015/11/30 20:57:53 127.0.0.1:49238: connection error: PROTOCOL_ERROR\r\n\tserver_test.go:413: Error while expecting a GOAWAY frame: read tcp 127.0.0.1:49238-\u003e127.0.0.1:49237: wsarecv: An existing connection was forcibly closed by the remote host.\r\nFAIL\r\nFAIL\tgolang.org/x/net/http2\t2.668s\r\n```\r\n\r\nI can reproduce this here. I don't know what the problem is.\r\n\r\nAlex",
	"user": {
		"login": "alexbrainman",
		"id": 9796621,
		"type": "User",
		"site_admin": false
	},
	"labels": [
		{
			"name": "OS-Windows"
		}
	],
	"comments": 8,
	"created_at": "2015-11-30T23:35:35Z",
	"updated_at": "2016-01-13T02:00:22Z",
	"milestone": {
		"id": 1067491,
		"number": 22,
		"title": "Unreleased"
	}
}
